### ISSUE TITLE
Number of training data

### ISSUE THREAD

**Issue by Jaclye** [yzshi5/GM-GANO#11]
1. Hi author I would like to know the minimum amount of training data needed for this model.
2. when my data is only a few hundred, can I make up for the lack of data by increasing the number of iterations
3. Is it normal for all four images to be in a downward trend?
<img width="1015" alt="image" src="https://github.com/yzshi5/GM-GANO/assets/167189645/81eb29f6-b2de-41bb-b546-40e525045c69">

**Comment by yzshi5** [comment]
Hi, the amount of samples in the training dataset varies from task to task, I recommend the training dataset should at least contain several thousand samples. 

Increasing the number of iterations wouldn't help given a small dataset. The loss function of GANO is not very interpretable, the only thing you need to care about is whether the loss converges.

**Comment by Jaclye** [comment]
Is this the state where the model training is complete？
<img width="993" alt="image" src="https://github.com/yzshi5/GM-GANO/assets/167189645/687e3b25-ab24-4219-886c-99bfcaa81ee2">

**Comment by yzshi5** [comment]
It seems the loss converges and the model is well-trained
